addons/binding/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/GardenaSmartImpl.java

```diff
@@ -342,9 +337,10 @@ public void sendCommand(Device device, GardenaSmartCommandName commandName, Obje
         }
 
         if (command != null) {
+            stopRefreshThread();
```

Comment:
The intermediate refresh was to fast, it was after 3 seconds. OK, i can do it in 6 seconds BUT, if the normal refreshThread runs in < 6 seconds, the values are wrong. 
e.g if i switch the watering computer valve to ON, it jumps to OFF and then again to ON within some seconds. That breaks rules with multiple watering computers.

OK, i removed the true parameter now